### DISCUSSION TITLE
[Feature Request] Option to display trailing whitespaces in row/column data

### DISCUSSION THREAD

**Discussion by Lautenschlager-id** [dbeaver/dbeaver#12481] [Q&A] [answered]
I would like to be able to see trailing whitespaces in data fields in my SQL tables.

For example, assume the value `test ` in the following query:
``insert into `player` ( `name` ) values ( "test " )``

This way I currently can't see the whitespace in "test", leading me to face some bugs in my application. It would be easier for me if DBeaver displayed the whitespace somehow.

It could be enabled in the settings menu, preferably.

**Comment by kseniiaguzeeva** [comment, marked as answer]
Thank you for the suggestion. We already have such option. You can enable it in Preferences>Editors>Text Editors
![dbeaver_mlKav90dEB](https://user-images.githubusercontent.com/51405061/118404532-00c45980-b67c-11eb-9336-43513313f9bb.png)

**Reply by Lautenschlager-id** [reply]
Thank you for answering!

Sadly, this is not what my requested aimed.

The setting mentioned above only displays whitespace characters in the SQL editor itself:
![image](https://user-images.githubusercontent.com/26045253/119515071-ed844e80-bd4b-11eb-9563-d06bf40d36e0.png)

My request was that whitespaces would be displayed in data fields:
![image](https://user-images.githubusercontent.com/26045253/119515155-fd039780-bd4b-11eb-9fef-51823b710c31.png)
There is a whitespace, as you can see below, that I would like to have noticed when reading its content. Perhaps with the same whitespace character from the setting you've mentioned.
![image](https://user-images.githubusercontent.com/26045253/119515189-042aa580-bd4c-11eb-98f9-47a7ea61de0c.png)

Would it be possible to move this discussion to an issue again?

Thanks in advance.

**Reply by kseniiaguzeeva** [reply]
Unfortunately no. Can I ask you to create a new FR for displaying spaces in Data Editor?
